CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 
 ### Improvements
 - [\#694](https://github.com/Manta-Network/Manta/pull/694) Switch to u128::MAX in fungible ledger transfer integration test.
+- [\#469](https://github.com/Manta-Network/Manta/pull/469) Switch runtime to wasm only.
```

Comment:
should be 701.

Reply:
Ahh...  How stupid im

Reply:
Don't worry about the changelog, just revert the whole change. We'll fill it out when we release because it's causing merge  conflicts non stop right now.